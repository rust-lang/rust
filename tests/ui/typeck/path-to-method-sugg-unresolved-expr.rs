fn main() {
    let page_size = page_size::get();
    //~^ ERROR failed to resolve: use of undeclared crate or module `page_size`
}
