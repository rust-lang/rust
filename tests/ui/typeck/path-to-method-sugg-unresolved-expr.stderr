error[E0433]: failed to resolve: use of undeclared crate or module `page_size`
  --> $DIR/path-to-method-sugg-unresolved-expr.rs:2:21
   |
LL |     let page_size = page_size::get();
   |                     ^^^^^^^^^ use of undeclared crate or module `page_size`

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0433`.
